src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1288,6 +1304,16 @@ public void watchFiles(File buildFile, File outputDirectory, File testOutputDire
                     Thread.sleep(pollingInterval);
                 }
             }
+        } finally {
+            if (watcher != null) {
+                try {
+                    watcher.close();
+                } catch (IOException e) {
+                    // TODO Auto-generated catch block
+                    e.printStackTrace();
```

Comment:
Log the exception using `error(.....)`

Reply:
How is this?
```
catch (IOException e) {
                    error("An error occurred attempting to close the file watcher. " + e.getMessage());
                }
```

Reply:
Also include the throwable as the second parameter so the stacktrace can be logged. e.g. use `error(String msg, Throwable e)`